charts/k8ssandra/values.yaml

```diff
@@ -35,8 +35,26 @@ cassandra:
   # -- Pull policy for the cassandra image
   #   pullPolicy: IfNotPresent
 
+  # -- Security context override for cassandra container
+  # K8ssandra defaults readOnlyRootFilesystem: true
+  # TODO - overriding to disable default for now, due to mgmt-api root privs required
+  securityContext:
+    readOnlyRootFilesystem: false
+
+  # -- Security context override for pod where Cassandra container resides
+  podSecurityContext:
+    readOnlyRootFilesystem: false
+
+  # -- Cassandra base init container
+  baseConfig:
+    # -- Security context override for base init container
+    # K8ssandra defaults readOnlyRootFilesystem: true
+    securityContext: {}
```

Comment:
Are there security settings that won't work with this container? If so we should either document it or not allow it. I'm fine with just documenting it and doing that in a follow PR if that works for you.